cmd/docs/generator.go

```diff
@@ -16,9 +16,9 @@ import (
 
 func main() {
 	var conf, doc, tmpl string
-	flag.StringVar(&conf, "conf", "schema.json", "Schema file")
-	flag.StringVar(&tmpl, "tmpl", "docs/dev/doc.tmpl", "Template file")
-	flag.StringVar(&doc, "doc", "docs/dev/doc.md", "Output file")
+	flag.StringVar(&conf, "conf", "../../schema.json", "Schema file")
+	flag.StringVar(&tmpl, "tmpl", "../../docs/dev/doc.tmpl", "Template file")
+	flag.StringVar(&doc, "doc", "../../docs/dev/doc.md", "Output file")
```

Comment:
Can't we move these files inside cmd/docs/ too ? :/

Reply:
thought to have doc.md in the documentation folder and it would be nice to have that seperated with the go code @stp had same thoughts on this.

Reply:
Oh okay :)

Reply:
I think both the template and the doc are easier for contributing people to edit closer to the root folder. It looks a bit ugly with the path, but from a contributor perspective it makes it easier.
Happy to have another solution so.